development/openstack-telemetry-bionic-queens/openrc

```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```

Comment:
This looks different too?  should it be a link?

Reply:
its fine, it is a link - to a file in the same directory